### PR TITLE
fix(sdk): do not use `previewFeatures` to determine `getDMMF` engine

### PR DESCRIPTION
**Pull request by williamluke4**
https://github.com/prisma/prisma/issues/6301#issuecomment-850710377

### CODE REVIEW

**Review by pimeys — CHANGES_REQUESTED** [review]

File: src/packages/sdk/src/engine-commands/getDmmf.ts [hunk 1]
```diff
@@ -33,10 +33,9 @@ export type GetDMMFOptions = {
 export async function getDMMF(options: GetDMMFOptions): Promise<DMMF.Document> {
   warnOnDeprecatedFeatureFlag(options.previewFeatures)
 
-  const useNapi =
-    process.env.PRISMA_FORCE_NAPI === 'true' ||
-    options.previewFeatures?.includes('nApi')
```
[the remark this answers is not in the document]
Comment (williamluke4) [reply src/packages/sdk/src/engine-commands/getDmmf.ts:38]: > Didn't this change to Napi?
No, it changed `nApi` 
> And is it correct to remove this check?
This is what was causing `Error: Invalid or unexpected token`

**Review by pimeys — APPROVED** [review]

File: src/packages/sdk/src/engine-commands/getDmmf.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (pimeys) [reply src/packages/sdk/src/engine-commands/getDmmf.ts:38]: Cool. Approved.

File: src/packages/sdk/src/engine-commands/getDmmf.ts [hunk 2]
```diff
@@ -70,6 +70,8 @@ async function getDmmfBinary(options: GetDMMFOptions): Promise<DMMF.Document> {
     EngineTypes.queryEngine,
     options.prismaPath,
   )
+  debug(`Using Query Engine Binary at: ${queryEnginePath}`)
```
Comment (Jolg42) [inline src/packages/sdk/src/engine-commands/getDmmf.ts:73]: Are these debug logs useful, they seem redundant, can we remove them? @williamluke4

Comment (Jolg42) [reply src/packages/sdk/src/engine-commands/getDmmf.ts:73]: Ok they are not redundant, It's just me and GitHub UI 👁️

**Pull request merged**